package.json

```diff
@@ -16,6 +16,25 @@
     "node": "5.10.x"
   },
   "dependencies": {
+    "bootstrap-sass": "^3.3.7",
```

Comment:
We are using caret and tilde in dependencies may be we should always use tilde? In my opinion automatic update to  the most recent major is not a good approach for dependencies. What is your opinion @patrys ?